service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -95,19 +97,14 @@ public void preparePipelineRun(
               .formatted(jobId));
     }
 
-    // hardcode for now, ultimately fetch from Rawls or store in config?
-    // this is for dev control workspace:
-    // https://bvdp-saturn-dev.appspot.com/#workspaces/teaspoons-imputation-dev/teaspoons_imputation_dev_control_workspace_20240726
-    String workspaceStorageContainerUrl = "gs://fc-secure-972057b6-fce1-4ec9-be0c-0e5c97fdc3e8";
```

Comment:
Im a dummy and forgot to do this in my PR but since you already created the empty beagle workspace, do you mind updating the SubmitCromwellSubmissionStep to use the `wdlMethodName` when creating the submission.